DQMOffline/Trigger/python/SiPixel_OfflineMonitoring_cff.py

```diff
@@ -2,8 +2,15 @@
 
 from DQMOffline.Trigger.SiPixel_OfflineMonitoring_Cluster_cff import *
 from DQMOffline.Trigger.SiPixel_OfflineMonitoring_TrackCluster_cff import *
+from RecoPixelVertexing.PixelLowPtUtilities.siPixelClusterShapeCache_cfi import *
+from DQM.SiPixelMonitorTrack.RefitterForPixelDQM import *
+
+hltSiPixelClusterShapeCache = siPixelClusterShapeCache.clone(src = 'hltSiPixelClusters')
+hltrefittedForPixelDQM = refittedForPixelDQM.clone(src ='hltMergedTracks')
 
 sipixelMonitorHLTsequence = cms.Sequence(
-    hltSiPixelPhase1ClustersAnalyzer
-    #+ hltSiPixelPhase1TrackClustersAnalyzer
+    hltSiPixelClusterShapeCache
+    + hltSiPixelPhase1ClustersAnalyzer
+    + refittedForPixelDQM
```

Comment:
shouldn't this be

```suggestion
    + hltrefittedForPixelDQM
```

?

Reply:
Yes, It should. Just push a commit to fix it.